server/docs/readMe.md

```diff
@@ -30,6 +30,10 @@ can be used by any role, creates new quote record at stage ‘requested’
 
 **PUT /quotes/{id}** - admin only, quote ‘id' is required, used for adding prices to requested quotes
 
+**PUT /quotes/{id}/statusTypeId={statusTypeId}** - admin only, quote 'id' is
+required, statusTypeId is required to be an integer within the set {1,4}, used
+for updating or creating statusTypeId of a quote
+
```

Comment:
In the interest of maintaining REST whenever possible, can we keep everything in the path? something like: /quotes/{quoteId}/status/{statusTypeId}